gnosis/safe/tests/api/test_transaction_service_api.py

```diff
@@ -183,3 +184,15 @@ def test_get_safe_transaction(self):
                 f"Cannot get transaction with safe-tx-hash={safe_tx_hash.hex()}:",
                 str(context.exception),
             )
+
+    def test_decode_data(self):
+        with patch.object(TransactionServiceApi, "_post_request") as mock_post_request:
+            data = HexStr(
+                "0x095ea7b3000000000000000000000000e6fc577e87f7c977c4393300417dcc592d90acf8ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
+            )
+            to_address = "0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C"
+            self.transaction_service_api.decode_data(data, to_address)
```

Comment:
You are not checking the function is returning anything. I could change `return response.json()` in `decode_data` function to `return None` at the end and this test will pass. You should also mock the result and check the function is returning what you expect.